src/chapters/fetch-json/exercises.rst

```diff
@@ -85,4 +85,181 @@ JSON
 Fetch
 -----
 
-.. todo:: add exercises for fetch
+#. Create a file called ``fetch_planets.html``.
+   In your terminal, type ``touch fetch_planets.html``. 
+
+#. Open the file in VSCode and add the preliminary HTML:
+
+   .. sourcecode :: html
+      :linenos:
+
+      <!DOCTYPE html>
+      <html>
+         <head>
+            <title>Fetch Planets</title>
+            <script>
+               window.addEventListener("load", function(){
+                  // TODO: fetch planets JSON
+               });
+            </script>
+         </head>
+         <body>
+            <h1>Destination</h1>
+            <div id="destination">
+               <h3>Planet</h3>
+            </div>
+         </body>
+      </html>
+
+#. The URL where our planet data is located is: ``"https://handlers.education.launchcode.org/static/planets.json"``.
+   Add the code to fetch this URL inside the load event listener on line 6:
+
+   .. sourcecode :: js
+      :linenos:
+
+      window.addEventListener("load", function() {
+         fetch("https://handlers.education.launchcode.org/static/planets.json").then(function(response){
+            // TODO: do something after fetching and receiving a response
+         });
+      });
+
+#. Peek the response returned in the request with a print statement:
+
+   .. sourcecode :: js
+      :linenos:
+
+      window.addEventListener("load", function() {
+         fetch("https://handlers.education.launchcode.org/static/planets.json").then(function(response){
+            console.log(response);
+         });
+      });
+
+   Copy the file path of this HTML file and paste it as the URL in your browser.
+   You won't see much on the page yet. Open your developer tools and examine both the 
+   *Console* tab for the response value, as well as the *Network* tab for the request status.
+
+#. Use the ``.json()`` method on your response now to see more of the data in the console:
+
+   .. sourcecode :: js
+      :linenos:
+
+      window.addEventListener("load", function() {
+         fetch("https://handlers.education.launchcode.org/static/planets.json").then(function(response){
+            response.json().then(function(json) {
+               console.log(json);
+            });
+         });
```

Comment:
Maybe just include lines 148-150 in the ``sourcecode`` directive. That way students who read the whole page ahead of time won't get a leg up.